glowing_waffles/coordinates.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function, division
+import numpy as np
+
+from ccdproc import CCDData
+from astropy.wcs import WCS
+
+def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, is_pix=True):
```

Comment:
Could you please also add an `origin` argument, that defaults to zero, and then pass that argument in to the conversion routines, as indicated below?